default.yml

```diff
@@ -7,6 +7,7 @@ require:
   - ./lib/update_not_update_attributes
 
 AllCops:
+  NewCops: enable
```

Comment:
I'm a bit afraid of https://www.pivotaltracker.com/story/show/161922910 or https://www.pivotaltracker.com/n/projects/2395661/stories/152606046 happening again if we do this. I guess it's fine as long as we don't end up with a huge rubocop todo file because there is definitely some value in us staying up to date with the rubocop community. 

@L-Burtness - thoughts?

Reply:
Yeah we'll probably get new rubocop todo items but I think it's worth it to be more up to date. If we do have new todo items, I can fix them in maji because this reminds me that I wanted to move some rules I disabled in the maji specific rubocop file to this shared gem anyway